src/pudl/output/ferc1.py

```diff
@@ -1007,11 +1045,16 @@ def boom(self, clean_xbrl_metadata_json: dict):
     def redefine_calculations_with_components_out_of_explosion(
         self, meta_explode: pd.DataFrame
     ) -> pd.DataFrame:
-        """Overwrite the calculations with calculation components not in explosion."""
-        calc_explode = convert_calculations_into_calculation_component_table(
-            meta_explode
-        )
-        calc_explode["in_explosion"] = calc_explode.source_tables.apply(
+        """Overwrite the calculations with calculation components not in explosion.
+
+        TODO: remove this method and the one call for it within boom once the
+        Tree/Forest situation directly uses the calculation components instead of the
+        embedded calculations.
+        """
```

Comment:
this step will no longer be necessary once the tree setup uses the calc components directly bc `calculation_components_explosion` now does this work